src/filtertransdialog.cpp

```diff
@@ -86,6 +86,7 @@ mmFilterTransactionsDialog::mmFilterTransactionsDialog(wxWindow* parent, int acc
     , payeeID_(-1)
     , refAccountID_(account_id)
     , refAccountStr_("")
+    , m_filterStatus("")
```

Comment:
‘mmFilterTransactionsDialog::m_filterStatus’ will be initialized after ‘double mmFilterTransactionsDialog::m_min_amount’ when initialized here [-Wreorder]